### ISSUE TITLE
General Feedback

### ISSUE THREAD

**Issue by rfazeli** [hyang0129/foodclassapp#2]
The repo is very well organized, I just have a few minor comments below. I will also pull the code later today to follow your instructions in README and run the code locally and will let you know my feedback.

1. You can rename `class_name` to `CLASS_NAME` since constant names are usually UPPERCASE
https://github.com/hyang0129/foodclassapp/blob/897be4834b7cdc905192d3238d88df6bee20c934/src/class_names.py#L3
2. Again the convention is to name constants as UPPERCASE. Maybe you can rename non-constants to be lowercase e.g. `MODEL_BUCKET` -> `model_bucket` etc.
https://github.com/hyang0129/foodclassapp/blob/897be4834b7cdc905192d3238d88df6bee20c934/src/model.py#L10 
3. You can keep FOODTRAINER.ipynb under a `notebooks/` directory for anyone who wants to see your work in an interactive environment. You should also break down this notebook into a few scripts and include them in the `src/` directory. The scripts could be 
    1. `config.py` to keep your hyperparameters
    2. `preprocess.py` to keep all your data processing and augmentation functions/steps
    3. `loss.py`
    4. `optimizer.py`
    5. `training.py`
